Aftonbladet.js

```diff
@@ -0,0 +1,209 @@
+{
+	"translatorID": "235c5c47-0c04-4d77-9afe-e22265a017a9",
+	"label": "Aftonbladet",
+	"creator": "Jonatan Svensson Glad",
+	"target": "^https?://ww*\.aftonbladet\.se",
```

Comment:
Okay, then I suggest to make this two choices more clear with something like `ww(w|c)`...